login_password_page.php

```diff
@@ -213,13 +213,12 @@
 				echo '<input type="hidden" name="install" value="true" />';
 			}
 
-			echo '<input type="hidden" name="username" value="', string_html_specialchars( $t_username ), '" />';
 
 			echo sprintf( lang_get( 'enter_password' ), string_html_specialchars( $t_username ) );
 
 			# CSRF protection not required here - form does not result in modifications
 			?>
-
+			<input hidden readonly type="text" name="username" class="hidden" tabindex="-1" value="<?php echo string_html_specialchars( $t_username ) ?>" id="hidden_username" />
```

Comment:
Is it not redundant to use both _hidden_ attribute and class ? Do we even need the class at all ?

Reply:
> Is it not redundant to use both hidden attribute and class ?

I'm not sure. I'd say the css style is provided as a fallback in case the property is not supported.
I'm open to change it if you think otherwise.

Reply:
According to https://caniuse.com/#feat=hidden, the _hidden_ attribute is not supported under IE10, and that's still on our [official client requirements list](http://mantisbt.org/docs/master/en-US/Admin_Guide/html-desktop/#admin.install.requirements.client), so let's go for belt & braces then :wink: